metadata-db-app/src/migrations/062_add_variable_constraint.clj

```diff
@@ -0,0 +1,54 @@
+(ns migrations.062-add-variable-constraint
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private table-name "cmr_variables")
+(def ^:private table (format "METADATA_DB.%s" table-name))
+(def ^:private new-constraint-name "variables_con_n_r_p")
+(def ^:private new-index-name "variables_idx_n_r_p")
+(def ^:private old-constraint-name "variables_con_rev")
+(def ^:private old-index-name "variables_ucr_i")
+
+(defn- add-variable-constraint
+  []
+  (h/sql
+   (format (str "ALTER TABLE %s "
```

Comment:
I actually like the string literal approach more readable. Since we already know the value of the names, we should just use a string literal here.